src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/ThirdPartyUser.razor

```diff
@@ -1,3 +1,73 @@
-﻿@inherits AdminCompontentBase
+﻿@using Masa.Stack.Components
```

Comment:
Move it to _Imports

Reply:
After this change, some components of Masa.Stack.Components have the same copy in Auth, and the global application of Masa.Stack.Components will conflict